hathor/prometheus.py

```diff
@@ -42,6 +44,50 @@
     'send_token_timeouts': 'Number of times send_token API has timed-out',
 }
 
+# Defines the metrics related to peer connections
+PEER_CONNECTION_METRICS = {
+    "received_messages": Gauge(
+        METRIC_PREFIX + "peer_connection_received_messages",
+        "Counts how many messages the node received from a peer",
+        labelnames=["network", "connection_string", "peer_id"],
```

Comment:
Looks like all `labelnames` are equal. Should we have a variable and reuse it?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/467/commits/fbc0453ccbcbed471d13cb3f66d7481a8666b8e7